client/messaging/connected-username.tsx

```diff
@@ -4,12 +4,15 @@ import { SbUserId } from '../../common/users/user-info'
 import { useAnchorPosition } from '../material/popover'
 import { ConnectedUserProfileOverlay } from '../profile/user-profile-overlay'
 import { useAppSelector } from '../redux-hooks'
+import { blue100 } from '../styles/colors'
 
-const Username = styled.span`
+const Username = styled.span<{ $isMention: boolean }>`
```

Comment:
Think you'd be better off passing through a `className` prop to this element and styling this how you want in the mention code. Otherwise every time we need a different style we'll have to add more props here, or a whole enum, etc. The typical styled-components pattern is more the subclassing thing (e.g. `styled(ConnectedUsername)`).